src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.Uno.targets

```diff
@@ -12,8 +12,8 @@
 		<_UnoProjectSystemPackageReference Include="SkiaSharp.Views.Uno.WinUI" ProjectSystem="true"/>
 	</ItemGroup>
 
-	<ItemGroup>
-		<_UnoProjectSystemPackageReference Include="Uno.WinUI.Svg" ProjectSystem="true" Condition="$(UnoFeatures.Contains(';Svg;')) AND $(IsUnoHead) == 'true'" />
+	<ItemGroup Condition="$(UnoFeatures.Contains(';Svg;')) AND $(IsUnoHead) == 'true' AND '$(IsBrowserWasm)' != 'true' ">
```

Comment:
Can't `Svg` be defined last without a trailing `;`?

Reply:
It's handled automatically by the prior definition of UnoFeatures, a `;` is added at the end.